lib/supa/commands/namespace.rb

```diff
@@ -2,10 +2,19 @@ module Supa
   module Commands
     class Namespace < Supa::Command
       def represent
+        return if hide?
+
         @tree[@name] = {}
+        return if value.nil? && empty_when_nil?
```

Comment:
I see that we don't even use with new option. I think that we should remove it only slows down the code. WDYT?

Reply:
I would keep it to make the interface of `Namespace` consistent with other commands.

Reply:
I see, wasn't aware that we even have `empty_when_nil` in the first place. Let's keep it then.